### ISSUE TITLE
Show the recommended badge in autocomplete results

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-frontend#8029]
Depends on ~~https://github.com/mozilla/addons-server/issues/11439~~
This is a spin-off from https://github.com/mozilla/addons-frontend/issues/7793.

As per https://github.com/mozilla/addons-frontend/issues/7793#issuecomment-490908595, we should show the recommended badge in the autocomplete results:

![image](https://user-images.githubusercontent.com/6309827/57457773-6c46e580-7270-11e9-80e5-63ea51101171.png)

**Comment by bobsilverberg** [comment]
@eviljeff I believe this will need a change to the [Autocomplete](https://addons-server.readthedocs.io/en/latest/topics/api/addons.html#autocomplete) endpoint in order to expose `is_recommended`.

**Comment by ioanarusiczki** [comment]
@bobsilverberg I verified the autocomplete on AMO dev - FF67(Win10). 
- the badge shows in the autocomplete results for all types of add-ons: `extension`, `language`, `dictionary`, `statictheme`.
- hovering (or navigating using the keyboard ) over an add-on with a recommended badge will display the blue arrow which is as expected.
- the default `sort` for autocomplete (as it's for search) is [`relevance`](https://addons-server.readthedocs.io/en/latest/topics/api/addons.html#addon-search-sort) if a search query is present so the recommended add-on doesn't have to be the first always (if I use a general term such as `Web` or `awesome` as in the example below). But in general, I found the recommended addon in the first place if I typed in the first part of an add-ons name that I knew it's a recommended one. 

![autocomplete](https://user-images.githubusercontent.com/33448286/58173867-20029900-7ca5-11e9-8f71-130ae5790491.gif)
